### PR TITLE
Link repair

### PR DESCRIPTION
**Pull request by kghamilton89**

### CODE REVIEW

**Comment by kghamilton89** [comment]
* Repair `.yml` and `.sh` links.
* **Should** removed from `/tutorials` and `/integration`.
* Style refactoring for `/calculated-values`, `/schema-based-parser-mod`, `/moving-avg`, and `/add-calculated-value`.
* Some text blocks contained `sg` instead of `sh` tag.
* Chicago Crime article had broken title photo.

File: integrations/itm/README.md [hunk 1]
```diff
@@ -80,7 +80,7 @@ Because statistics from ITM agents are received by ATSD with no delay, use this
 
 ### Configure `inotify` Script to Read CSV files and Upload into ATSD
 
-* Download [`inotify_sender`](inotify_sender.sh) script to your WPA server.
+* Download [`inotify_sender`](https://github.com/axibase/atsd-use-cases/blob/master/integrations/itm/inotify_sender.sh) script to your WPA server.
```
Comment (rodionos) [inline integrations/itm/README.md]: Use relative links per https://github.com/axibase/docs-util/blob/master/guide.md#links

File: integrations/kafka/README.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ This document describes how to monitor availability and performance of [Apache K
 3. Import the [`kafka-jmx`](resources/job_jmx_kafka-jmx.xml) job.
 4. Locate the newly-imported `kafka-jmx` job on the **Jobs** page.
 5. On the **JMX Job** page, enable the job status by setting the flag in the **Enabled** checkbox.
-6. Adjust the cron expression if required. For more information on cron expressions, see [Scheduling](https://github.com/axibase/axibase-collector/blob/master/scheduling.md).
+6. Adjust the `cron` expression if required. For more information on `cron` expressions, see [Scheduling](https://github.com/axibase/axibase-collector/blob/master/scheduling.md).
```
Comment (rodionos) [inline integrations/kafka/README.md]: Link to a corresponding html file in /docs/axibase-collector

File: integrations/kafka/README.md [hunk 3]
```diff
@@ -75,7 +75,7 @@ Consumer offset is collected using a Kafka console consumer reading events from
 
 Log in to the Kafka server.
 
-Download the [script](resources/send_offset.sh) into Kafka `bin` directory.
+Download the [script](https://github.com/axibase/atsd-use-cases/blob/master/integrations/kafka/resources/send_offset.sh) into Kafka `bin` directory.
```
Comment (rodionos) [inline integrations/kafka/README.md]: Use relative url

File: integrations/socrata/python/README.md [hunk 4]
```diff
@@ -15,10 +15,10 @@ For this particular exercise, import the data on the [New York State Solar Photo
 ### Download the `docker-compose.yml` file
 
 ```sh
-curl -o docker-compose.yml https://raw.githubusercontent.com/axibase/atsd-use-cases/master/socrata/python/docker-compose.yml
+curl -o docker-compose.yml https://raw.githubusercontent.com/axibase/atsd-use-cases/master/integrations/socrata/python/docker-compose.yml
 ```
 
-The [`docker-compose.yml`](docker-compose.yml) file configures Axibase Time Series Database (ATSD) and Axibase Collector services along with the required dependencies. The role of the Collector is to automatically download and parse a specific data.gov JSON file while ATSD serves as the SQL-enabled database.
+The [`docker-compose.yml`](https://github.com/axibase/atsd-use-cases/blob/master/integrations/socrata/python/docker-compose.yml) file configures Axibase Time Series Database (ATSD) and Axibase Collector services along with the required dependencies. The role of the Collector is to automatically download and parse a specific data.gov JSON file while ATSD serves as the SQL-enabled database.
```
Comment (rodionos) [inline integrations/socrata/python/README.md]: Use relative url. Use full URL only when we refer to files in launch commands.

File: integrations/zookeeper/README.md [hunk 5]
```diff
@@ -39,7 +39,7 @@ Import the [`zookeeper-jmx`](resources/jobs.xml) job.
 
 Locate the `zookeeper-jmx` job in the **Jobs** list.
 
-Adjust the cron expression if required. For more information on cron expressions, see [Scheduling](https://github.com/axibase/axibase-collector/blob/master/scheduling.md).
+Adjust the `cron` expression if required. For more information on `cron` expressions, see [Scheduling](https://github.com/axibase/axibase-collector/blob/master/scheduling.md).
```
Comment (rodionos) [inline integrations/zookeeper/README.md]: Link to docs/axibase-collector

File: integrations/itm/README.md (hunk 1 above)
Comment (kghamilton89) [reply integrations/itm/README.md]: relatives used

File: integrations/kafka/README.md (hunk 2 above)
Comment (kghamilton89) [reply integrations/kafka/README.md]: correct docs links

File: integrations/kafka/README.md (hunk 3 above)
Comment (kghamilton89) [reply integrations/kafka/README.md]: here as well

File: integrations/socrata/python/README.md (hunk 4 above)
Comment (kghamilton89) [reply integrations/socrata/python/README.md]: changed as discussed

File: integrations/zookeeper/README.md (hunk 5 above)
Comment (kghamilton89) [reply integrations/zookeeper/README.md]: changed.

**Comment by kghamilton89** [comment]
* Collector Docs properly referenced.
* Relatives used for `.sh` and `.yml` files.
* `resources` directories created as needed.

**Pull request merged**